src/main/java/sg/ncl/MainController.java

```diff
@@ -1576,15 +1595,18 @@ public String experiments(Model model, HttpSession session) {
                 for (int k = 0; k < experimentsArray.length(); k++) {
                     Experiment2 experiment2 = extractExperiment(experimentsArray.getJSONObject(k).toString());
                     Realization realization = invokeAndExtractRealization(experiment2.getTeamName(), experiment2.getId());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this useless assignment to local variable "error". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "error" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)